src/test/resources/pages/instructorSearchPageSearchComments.html

```diff
@@ -105,7 +105,7 @@
               <span class="text-muted">
                 To
                 <b>
-                  student1 In Course1 (Team 1.1, searchUI.student1InCourse1@gmail.tmt)
+                  student1 In Course1\u003c/option\u003e\u003c/td\u003e\u003c/div\u003e\u0027\" (Team 1.1\u003c/td\u003e\u003c/div\u003e\u0027\", searchUI.student1InCourse1@gmail.tmt)
```

Comment:
We can fix it later. In Search Page, we only have to show search results so this bug only affect frontend. It is not critical.